baselines/dom.generated.d.ts

```diff
@@ -3796,7 +3796,12 @@ declare var FocusNavigationEvent: {
 }
 
 interface FormData {
-    append(name: any, value: any, blobName?: string): void;
+    append(name: string, value: any, fileName?: string): void;
```

Comment:
That is correct, it's just a `type` so it should not be in the generated code, but it should be available to users to use as a type.